pkgs/games/sgfutils/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, openssl
+, iconv
+, makeWrapper
+, imagemagick
+, makeFontsConf
+}:
+stdenv.mkDerivation
+{
+  pname = "sgfutils";
+  version = "unstable-2017-11-27";
+  buildInputs = [ openssl makeWrapper ] ++ lib.optionals stdenv.isDarwin [ iconv ];
+  src = fetchFromGitHub {
+    owner = "yangboz";
+    repo = "sgfutils";
+    rev = "11ab171c46cc16cc71ac6fc901d38ea88d6532a4";
+    hash = "sha256-KWYgTxz32WK3MKouj1WAJtZmleKt5giCpzQPwfWruZQ=";
+  };
+  buildPhase = "make all";
```

```suggestion
  buildPhase = ''
    runHook preBuild
    make all
    runHook postBuild
  '';
```

When overriding a standard phase, make sure to call the pre and pots hooks. Same review for `installPhase`.